### ISSUE TITLE
Full-screen compose box

### ISSUE THREAD

**Issue by alya** [zulip/zulip-mobile#4873]
At present, the compose box is quite small on mobile. Especially with Zulip's recommended style of slightly longer messages, it would be very helpful to have a full-size compose box option for composing long messages.

The toggle between the small vs. full-size compose box can be double arrows pointing in or out in the corner, a la Slack.

**Comment by SilentCruzer** [comment]
Can I work on this issue?

**Comment by alya** [comment]
@gnprice can commend on how approachable this issue is.

**Comment by SilentCruzer** [comment]
(chat continued on [CZO](https://chat.zulip.org/#narrow/stream/48-mobile/topic/Expand.20compose.20box/near/1261834))

**Comment by SilentCruzer** [comment]
@alya , should I add the expoand icon inside the text-box like this:

![expand](https://user-images.githubusercontent.com/73246484/136096973-83e73c4a-e71b-47c6-88e1-0000e2504f08.png)

The only problem with this is that we need to add padding to the left of the icon, so text-overflow doesn't happen.

**Comment by alya** [comment]
@SilentCruzer Yeah, I think that's the right place for it. I think it could be closer to the right edge of the text box in your screenshot.

**Comment by alya** [comment]
@sumj25 please discuss with @SilentCruzer whether they are still working on this issue.

**Comment by SilentCruzer** [comment]
@alya, I have added the expand feature, this is how it looks now:

https://user-images.githubusercontent.com/73246484/137984611-f41bbdba-9a0a-492f-8359-103b48197974.mp4

Can you suggest changes if any?

**Comment by alya** [comment]
We should just use different gray tones in light vs dark mode, as we do with many UI elements.
